appveyor.yml

```diff
@@ -1,76 +1,42 @@
-# https://ci.appveyor.com/project/OpenWaterAnalytics/stormwater-management-model
-
+version: 2.0.{build}
+
+image:
+- Visual Studio 2013
+- Visual Studio 2015
+- Visual Studio 2017
```

Comment:
Please include

```
# - Visual Studio 2008
- Visual Studio 2010
```

The comment for 2008 is a reminder of it not working out of the box for python 2.7

So it could be 
```
image:
# - Visual Studio 2008  # For working with python 2.7
- Visual Studio 2010  # For working with python 3.4
- Visual Studio 2013
- Visual Studio 2015  # For working with python 3.5 and 3.6
- Visual Studio 2017
```

Reply:
@bemcdonnell then you need to use env matrix like

```
environment:
  matrix:
#  - vsver: 2008 arch: Win32
#  - vsver: 2008 arch: Win64
  - VS_VERSION: 2010 arch: Win32
  - VS_VERSION: 2010 arch: Win64
  - VS_VERSION: 2013 arch: Win32
  - VS_VERSION: 2013 arch: Win64
  - VS_VERSION: 2015 arch: Win32
  - VS_VERSION: 2015 arch: Win64
  - VS_VERSION: 2017 arch: Win32
  - VS_VERSION: 2017 arch: Win64
```

And then do something like
```
if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2013" and "%VS_VERSION%"=="2010" ....
```